src/plan/generational/global.rs

```diff
@@ -14,15 +14,18 @@ use crate::util::heap::VMRequest;
 use crate::util::metadata::side_metadata::SideMetadataSanity;
 use crate::util::metadata::side_metadata::SideMetadataSpec;
 use crate::util::options::Options;
+use crate::util::statistics::counter::EventCounter;
 use crate::util::ObjectReference;
 use crate::util::VMWorkerThread;
 use crate::vm::VMBinding;
 use std::sync::atomic::AtomicBool;
 use std::sync::atomic::Ordering;
-use std::sync::Arc;
+use std::sync::{Arc, Mutex};
 
 use mmtk_macros::PlanTraceObject;
 
+const WORST_CASE_COPY_EXPANSION: f64 = 1.5;
```

Comment:
Hmm... Yes I think you're right. In that case is 1.5x correct for OpenJDK and our other VMs? I can add it to `vm::ObjectModel`.

Reply:
Yeah. 1.5 should fine. For OpenJDK, 1 is probably fine (I assume we do not grow object size during copying for OpenJDK). I don't know how much this would affect performance. If it is significant, we may want to make 1 as the default value, and change the value for JikesRVM to 1.5.

Reply:
I've moved it to `ObjectModel`, but have kept the value to still be 1.5. I think OpenJDK technically doesn't change the size of copied objects, but I'm not certain so I'm being conservative currently.